libraries/plugins/chain/chain_plugin.cpp

```diff
@@ -327,6 +328,9 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
          ("checkpoint,c", bpo::value<vector<string>>()->composing(), "Pairs of [BLOCK_NUM,BLOCK_ID] that should be enforced as checkpoints.")
          ("flush-state-interval", bpo::value<uint32_t>(),
             "flush shared memory changes to disk every N blocks")
+#ifdef ENABLE_MIRA
+         ("indices-memory-replay", bpo::value<vector<string>>()->multitoken()->composing(), "Specify which indices should be in memory during replay")
```

Comment:
If this option exists only in the config file (which I agree it should) then you can prefix it with `memory`. I think `memory-replay-indices` sounds a bit better. A little less "yoda"-ish.

Reply:
I believe appbase will copy any configuration options to command line options. As it stands this option is available in both as per `steemd -h` and local testing. I'm not aware of a way to avoid this, please advise.

Reply:
You are correct. Config options are copied to command line arguments but not the other way.

Go ahead and name it `memory-replay-indices` and change `memory-replay` to `memory-replay,m`. I believe that will force the shorthand as `-m`, which is arguably no more arbitrary than `--memory` compared to `--memory-replay`.